ci/unit_tests/test_spleen_ct_segmentation.py

```diff
@@ -0,0 +1,120 @@
+        "bundle_root": "models/spleen_ct_segmentation",
+        "images": "$list(sorted(glob.glob(@dataset_dir + '/image_*.nii.gz')))",
+        "labels": "$list(sorted(glob.glob(@dataset_dir + '/label_*.nii.gz')))",
+        "epochs": 1,
+        "train#dataset#cache_rate": 0.0,
+        "validate#dataset#cache_rate": 0.0,
+        "train#dataloader#num_workers": 1,
+        "validate#dataloader#num_workers": 1,
+        "train#random_transforms#0#spatial_size": [32, 32, 32],
+    }
+]
+
+TEST_CASE_2 = [  # inference
+    {
+        "bundle_root": "models/spleen_ct_segmentation",
+        "datalist": "$list(sorted(glob.glob(@dataset_dir + '/image_*.nii.gz')))",
+        "handlers#0#_disabled_": True,  # do not load weights
+    }
+]
+
+
+class TestSpleenCTSeg(unittest.TestCase):
+    def setUp(self):
+        self.dataset_dir = tempfile.mkdtemp()
+        self.produce_fake_dataset()
+
+    def produce_fake_dataset(self):
+        dataset_size = 10
+        input_shape = (64, 64, 64)
+        for s in range(dataset_size):
+            test_image = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+            test_label = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+            image_filename = os.path.join(self.dataset_dir, f"image_{s}.nii.gz")
+            label_filename = os.path.join(self.dataset_dir, f"label_{s}.nii.gz")
+            nib.save(nib.Nifti1Image(test_image, np.eye(4)), image_filename)
+            nib.save(nib.Nifti1Image(test_label, np.eye(4)), label_filename)
+
+    def tearDown(self):
+        shutil.rmtree(self.dataset_dir)
+
+    @parameterized.expand([TEST_CASE_1])
+    def test_train_eval_config(self, override):
+        override["dataset_dir"] = self.dataset_dir
+        bundle_root = override["bundle_root"]
+        train_file = os.path.join(bundle_root, "configs/train.json")
+        eval_file = os.path.join(bundle_root, "configs/evaluate.json")
+
+        trainer = ConfigWorkflow(
+            workflow="train",
+            config_file=train_file,
+            logging_file=os.path.join(bundle_root, "configs/logging.conf"),
+            meta_file=os.path.join(bundle_root, "configs/metadata.json"),
+            **override,
+        )
+        trainer.initialize()
+        # check required and option properties
+        check_result = trainer.check_properties()
+        if check_result is not None and len(check_result) > 0:
+            raise ValueError(f"check properties for train config failed: {check_result}")
+        trainer.run()
+        trainer.finalize()
+
+        validator = ConfigWorkflow(
+            workflow="eval",
+            config_file=[train_file, eval_file],
+            logging_file=os.path.join(bundle_root, "configs/logging.conf"),
+            meta_file=os.path.join(bundle_root, "configs/metadata.json"),
+            **override,
+        )
+        validator.initialize()
+        # no need to check properties since evaluate.json overrides train.json
```

Comment:
Please check the properties again, `evaluate.json` overriding the config may remove necessary properties.